### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Evgeniy994**

### CODE REVIEW

File: app/managers.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self):
+        self._connection = sqlite3.connect("cinema.db3")
+        self.table_name = "actor"
+
+    def create(self, first_name: str, last_name: str):
+        self._connection.execute(
```
Comment (darten360) [inline app/managers.py:12]: It's a bad style to use first paranthese near word and second one on a new line

File: app/managers.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            f"INSERT INTO {self.table_name} "
+            "(first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self):
+        actor_cursor = self._connection.execute(
```
Comment (darten360) [inline app/managers.py:20]: the same

File: app/managers.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            f"SELECT * FROM {self.table_name}"
+        )
+
+        return [Actor(*row) for row in actor_cursor]
+
+    def update(
+            self, id_to_update: int, new_first_name: str, new_last_name: str
+    ):
+        self._connection.execute(
```
Comment (darten360) [inline app/managers.py:29]: I think you understood me

**Review by darten360 — CHANGES_REQUESTED** [review]

File: app/managers.py (hunk 3 above)
Comment (Evgeniy994) [reply app/managers.py:29]: looks like done)

**Review by Magus6784 — APPROVED** [review]

**Review by MasterpieceElbow — CHANGES_REQUESTED** [review]
Remove unnecessary file `identifier.sqlite` from the pull request

**Pull request closed without merging**